bot.py

```diff
@@ -611,8 +625,8 @@ async def clear_queue(self, message, _command_content):
         """
         Stop current song and remove everything from queue
         """
-        while not self.media_queue.empty():
-            self.media_queue.get()
+        while len(self.media_deque) != 0:
```

Comment:
```suggestion
        while len(self.media_deque) > 0:
```

Reply:
Why is this better?

Reply:
I think it's prettier. Functionally it's the same.

Reply:
I disagree. Checking whether the deque is empty would be:
```python
len(deque) == 0
```
So I think `!=` is a more clear negation.